taqueria-plugin-ligo/index.js

```diff
@@ -1,6 +1,5 @@
 const {Plugin, Task, Option} = require('@taqueria/node-sdk')
```

Comment:
On my machine, when I try to compile a contract `/contracts/counter.jsligo` with the Taq and ligo plugin built from this branch, I get the following error:
```
❯ taq compile                        
{
  errCode: 'E_UNEXPECTED',
  message: 'The plugin encountered a fatal error',
  previous: yn [ZodError]: [
    {
      "code": "invalid_type",
      "expected": "string",
      "received": "undefined",
      "path": [
        "task"
      ],
      "message": "Required"
    }
  ]
      at $h (/Users/houstonkrohman/code/taqueria/taqueria-sdk/index.js:88:14289)
      at rn.safeParse (/Users/houstonkrohman/code/taqueria/taqueria-sdk/index.js:88:16555)
      at rn.parse (/Users/houstonkrohman/code/taqueria/taqueria-sdk/index.js:88:16234)
      at cl (/Users/houstonkrohman/code/taqueria/taqueria-sdk/index.js:88:61553)
      at /Users/houstonkrohman/code/taqueria/taqueria-sdk/index.js:90:3980 {
    issues: [ [Object] ],
    format: [Function (anonymous)],
    addIssue: [Function (anonymous)],
    addIssues: [Function (anonymous)]
  }
}
```

Reply:
Looks like this is global to all plugins at this time